src/extension.ts

```diff
@@ -1132,7 +1132,13 @@ export async function activate(context: vscode.ExtensionContext) {
 
   registerIDFCommand("espIdf.apptrace", () => {
     PreCheck.perform([webIdeCheck, openFolderCheck], async () => {
-      if (appTraceTreeDataProvider.appTraceButton.label.match(/start/gi)) {
+      const appTraceLabel =
+        typeof appTraceTreeDataProvider.appTraceButton.label === "string"
+          ? appTraceTreeDataProvider.appTraceButton.label.match(/start/gi)
+          : appTraceTreeDataProvider.appTraceButton.label.label.match(
+              /start/gi
+            );
```

Comment:
is there a change in the API?

Reply:
Yes, on [vscode November 2020](https://code.visualstudio.com/updates/v1_52#_markdown-tree-tooltip-api) the TreeItem API changed tooltip and label from string to `string | TreeItemLabel ` and `string | MarkdownString`